.github/workflows/deploy_native.yml

```diff
@@ -0,0 +1,63 @@
+name: Build and Deploy Raspberry Pi OS Image
+
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: ["Build and Deploy Images"]
+    types:
+      - completed
+
+jobs:
+  deploy-pi-img:
+    runs-on: self-hosted
+    # The runner for this job is a Raspberry Pi OS Bullseye.
+    # Just install docker (curl -sSL https://get.docker.com/ | sh) and follow the instructions for setting up a new runner in
+    # https://github.com/bluerobotics/BlueOS-docker/settings/actions/runners/new
+
+    strategy:
+      matrix:
+        platforms: ["linux/arm/v7"]
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+
+      -
+        name: Set env NEED
+        run: |
+          if [[ ( '${{github.event_name}}' == 'workflow_dispatch' || '${{github.event.workflow_run.conclusion}}' == 'success' )
+            && (${GITHUB_REF} == refs/tags/* || ${GITHUB_REF} == refs/heads/*)
+          ]]; then
+              echo "NEED=true" >> "$GITHUB_ENV"
+          else
+              echo "NEED=false" >> "$GITHUB_ENV"
+          fi
+      -
+        name: Pimod Build
+        if: env.NEED == 'true'
+        # We use our own pimod as upstream doesn't provide armv7 images
+        run: |
+          wget https://raw.githubusercontent.com/williangalvani/pimod/master/pimod.sh && chmod +x pimod.sh
```

Comment:
We should open a PR in pimod and request for armv7.

Reply:
idk about that. we are kind of mis-using it from what I understand. what it does here could be done by chroot itself. I chose to keep it because we can work on top of it to get amd64 working, plus the PiFile is extremely useful.